.eslintrc.json

```diff
@@ -33,6 +33,8 @@
     "import/no-extraneous-dependencies": 1,
     "import/no-unresolved": "off",
 
+    "no-underscore-dangle": ["error", { "enforceInMethodNames": false, "allowAfterThis": true }],
```

Comment:
Completely optional but it suppresses this in my ide:
![image](https://user-images.githubusercontent.com/46766462/133482993-aa479036-095d-409d-b93e-e5527f903c99.png)

Reply:
Ooh, good catch.

Looks like this file the base airbnb and recommended `jsdoc` and `react` lint configurations.

We probably should update to using NC's configurations like we do in other repos:
https://github.com/BLC/ux-framework/blob/77a63d84396570858623a60d407de0260c686c50/.eslintrc.json#L2-L7

If this gets gross from doing that switch, understand if you don't want to include in this PR.

Reply:
I think this will also resolve the `missing returns description` lint warning happening at the bottom of the PR.

Reply:
I think the reason we haven't done this is that the `@nextcapital/eslint-config...` configurations are in the internal npm registry and this repo is public.